src/tracks/InstrumentTrack.cpp

```diff
@@ -1130,6 +1156,37 @@ void InstrumentTrackView::midiConfigChanged()
 
 
 
+QMenu * InstrumentTrackView::createFxMenu(QString title, QString newFxLabel)
```

Comment:
I see your point.  Otherwise you'd likely need to extend QMenu with a new FXAssignementMenu and store a reference to the parent... although it does seem weird when you find yourself asking the coat for a bunch of hangers. :)